_data-prepper/pipelines/configuration/sinks/s3.md

```diff
@@ -44,10 +43,10 @@ Use the following options when customizing the `s3` sink.
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
 `bucket` | Yes | String | The object from which the data is retrieved and then stored. The `name` must match the name of your object store.
-`codec` | Yes | [Buffer type](#buffer-type) | Determines the buffer type. 
+`codec` | Yes | [Codec](#codec) | The format of the output data.
 `aws` | Yes | AWS | The AWS configuration. See [aws](#aws) for more information.
 `threshold` | Yes | [Threshold](#threshold-configuration) | Configures when to write an object to S3. 
-`object_key` | No | Sets the `path_prefix` and the `file_pattern` of the object store. Defaults to the S3 object `events-%{yyyy-MM-dd'T'hh-mm-ss}` found inside the root directory of the bucket.
+`object_key` | No | Sets the `path_prefix` and the `file_pattern` of the object store. File pattern is always `events-%{yyyy-MM-dd'T'hh-mm-ss}` found inside the root directory of the bucket.  `path_prefix` is configurable.
```

Comment:
So I couldn't tell if this was on purpose or not but checking the code the parameter is not configurable (even though the old docs made it look configurable).